cg-spec/editors_draft.html

```diff
@@ -462,7 +462,53 @@ <h2>Discussion</h2>
     </section>
   </section>
 
-    <section id="n-triples-star">
+  <section id="trig-star">
+    <h2>TriG-star</h2>
+
+    <p>This section describes TriG-star,
+      a minimal extension of the TriG format [[TRIG]]
+      using the same production updates described in <a href="#turtle-star" class="sectionRef"></a>.</p>
+
+    <p class="note">As RDF-star describes <a>embedded triples</a>,
+      which are not necessarily described within any specific graph,
+      either named or default.</p>
```

Comment:
This sentence is not complete.

Reply:
It is complete, short of a possibly missing comma, but I’ll reword to make the intent clearer.